src/components/AddressComponent/AddressComponent.tsx

```diff
@@ -0,0 +1,52 @@
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+import { contractAddress } from 'config';
+import {
+  ACCOUNTS_ENDPOINT,
+  getExplorerLink,
+  MvxCopyButton,
+  MvxTrim,
+  useGetNetworkConfig
+} from 'lib';
+
+interface ContractAddressPropsType {
+  address: string;
+  isHeader?: boolean;
+}
+
+export const AddressComponent = ({
+  address,
+  isHeader = false
+}: ContractAddressPropsType) => {
+  const { network } = useGetNetworkConfig();
+  const explorerAddress = network.explorerAddress;
+  const explorerLink = getExplorerLink({
+    to: `/${ACCOUNTS_ENDPOINT}/${contractAddress}`,
+    explorerAddress
+  });
+  return (
+    <>
+      <p className='text-primary transition-all duration-300 font-normal text-xs xxs:text-sm pt-1 lg:pt-0 overflow-auto'>
```

Comment:
We need to migrate all these classes to use `@apply` so it's easier to read. Long term task, not for now. Just to keep in mind.